setup.py

```diff
@@ -89,6 +110,14 @@ def findRequirements():
   requirementsPath = os.path.join(REPO_DIR, "requirements.txt")
   requirements = parse_file(requirementsPath)
 
+  if nupicBindingsPrereleaseInstalled():
+    # User has a pre-release version of nupic.bindings installed, which is only
```

Comment:
We will not be releasing any `nupic.bindings` versions with the `.dev0` suffix. Only public releases will go on Pypi.